product_brand/models/res_config_settings.py

```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    module_product_brand_print = fields.Boolean()
```

Comment:
This may cause problems during migration, we don't guarantee that both modules will be migrated at the same time.

Reply:
Hi @sbejaoui thanks for your time!! I committed your suggestions, but what do you think about this one? I could remove it and make product_brand_sale_report a single module, or add in product_brand the dependence to product_brand_sale_report!
Let me know! Thanks

Reply:
Let's keep the two modules separate.

Reply:
Done @sbejaoui thanks